src/web/extractors.rs

```diff
@@ -445,7 +445,12 @@ impl BsoParam {
         // path: "/1.5/{uid}/storage/{collection}/{bso}"
         let elements: Vec<&str> = uri.path().split('/').collect();
         let elem = elements.get(3);
-        if elem.is_none() || elem != Some(&"storage") || elements.len() != 6 {
+        if elem.is_none() {
+            return Ok(Self { bso: "".to_owned() });
+        }
+        if elem != Some(&"storage")
```

Comment:
I think this one is to quiet the warning log messages on the other hand?

Reply:
Same here, some weird test behavior. I'd rather understand why the tests are expecting to be able to do what they do, or fix the tests so they don't do it if it doesn't make sense any more.

Reply:
which tests?

Reply:
The python tests. They do a DELETE that the rust code doesn't seem to expect.

Reply:
Can you double check this? I can't reproduce it over here

Reply:
Double checked, the way the code was before passes the tests. Reverted that bit.